### DISCUSSION TITLE
add support for singularity

### DISCUSSION THREAD

**Discussion by zhan4429** [TheBrownLab/PhyloFisher#78] [General]
In the future updates, can you add support for singularity?

**Comment by atice** [comment]
Hi Yucheng,

Sorry for the confusion. Could you please elaborate on you request so we could consider how best to support your request in future updates? Thank you for your interest in PhyloFisher!

Alex

Alexander K. Tice, Ph.D.
Department of Biological Sciences
Mississippi State University
027 Harned Hall
Lab Telephone: 662-325-4008

________________________________
From: Yucheng Zhang ***@***.***>
Sent: Thursday, April 27, 2023 6:53 PM
To: TheBrownLab/PhyloFisher ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [TheBrownLab/PhyloFisher] add support for singularity (Issue #77)


In the future updates, can you add support for singularity?

**Comment by zhan4429** [comment]
Hi Alex,

Thanks for the quick response. I mean in your smk files, you only support conda environment, like below:

    conda:
        'prequal.yaml'

I am wondering whether you can support singularity, like this:

    singularity:
       ''docker://XXXX"

Best,
Yucheng

**Comment by atice** [comment]
Hi Yucheng,

Thank you for you're comments. Another PhyloFisher team member clarified. We will certainly consider this and get back to you. Again, thank you for considering PhyloFisher for your work.

Alex

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by robert-ervin-jones** [comment]
Hi Yucheng,

A Docker container for PhyloFisher can be found at quay.io/biocontainers/phylofisher.

Let me know if you have any more questions.

Best,
Robert
